### ISSUE TITLE
Feature Request: Add CLI args

### ISSUE THREAD

**Issue by Liam-Deacon** [tromgy/swagger-yaml-to-json-schema#31]
`ytoj` works great when working in an interactive console, but I can't automate the conversion.

For instance I can't do `echo -e "input.yaml\noutput.json\n\n\ny\ny\n2" | ytoj`.

It would be great to add a command line interface so that it is possible to do things like the following without needing the user to provide input:

```bash
ytoj -i input.yaml -o output.json --dereference --additional-properties --indent=2
```

Is this something that could easily be done?

**Comment by tromgy** [comment]
Have you tried using the tool with the configuration saved in **ytoj.json** in your scripts?

Yes, the _first_ time the tool runs interactively, but if you choose to save the configuration, the next time it will not require user input.

**Comment by tromgy** [comment]
This functionality is now available in published version 4.0.0
